navigator/backend/src/main/scala/com/digitalasset/navigator/console/Pretty.scala

```diff
@@ -216,7 +216,10 @@ object Pretty {
   def argument(arg: model.ApiValue): PrettyNode = arg match {
     case model.ApiRecord(id, fields) =>
       PrettyObject(
-        fields.map(f => PrettyField(f.label, argument(f.value)))
+        fields.toSeq.zipWithIndex.map {
```

Comment:
Do you think it would make sense here to use `iterator` rather than `toSeq` and then `breakOut` at the end to not have and extra loop just to `zipWithIndex`?

Reply:
5ce7df5d46567bdf789c19155dbd3911b1a45730